### ISSUE TITLE
help me bro

### ISSUE THREAD

**Issue by minipapergarden** [frederikme/Botnet#1]
shell command output doesn't appear. just saying "pending <command>"

**Comment by frederikme** [comment]
Can you provide some screenshots?

**Comment by xroot000** [comment]
yes, shell not working (typed 'whoami' for example - returns nothing),  but 'calc' runs calc.exe - ok
